cmd/skupper/skupper.go

```diff
@@ -852,10 +852,16 @@ func init() {
 	cmdVersion := NewCmdVersion(newClient)
 	cmdDebugDump := NewCmdDebugDump(newClient)
 
+	//backwards compatibility commands hidden
+	cmdBind.Hidden = true
+	cmdUnbind.Hidden = true
+
```

Comment:
Suggest setting cmdBind.Deprecated and cmdUnbind.Deprecated strings to indicate usage for Service sub-commands. Such as:

Command "bind" is deprecated, Use 'service bind' command instead.